### PR TITLE
Plots: mention the VS Code Extension for visualization

### LINKED ISSUE #3717: Add references to VS Code for DVC plots
### Report

Some places where it would help:

https://dvc.org/doc/command-reference/plots

This might be odd in a CLI ref, but for example, the current docs say:
 
> DVC generates plots as static HTML webpages that can be open with a web browser. They can also be saved as SVG or PNG image files from the browser.

Seems like it would be useful to at least mention that they can be visualized in VS Code, since it's a better experience than the static HTML.

https://dvc.org/doc/start/data-management/metrics-parameters-plots

https://dvc.org/doc/start/experiments/visualization

@jorgeorpinel @shcheklein Thoughts?

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Integrated mentions of the VS Code Extension in

- [x] https://dvc.org/doc/command-reference/plots
- [x] https://dvc.org/doc/start/data-management/metrics-parameters-plots
- [x] https://dvc.org/doc/start/experiments/visualization

> On top of existing mentions in 1) https://dvc.org/doc/install/ide-plugins, 2) https://dvc.org/doc/user-guide/what-is-dvc#core-features, and 3) https://dvc.org/doc/user-guide/experiment-management/experiments-overview#work-with-dvc-experiments-from-a-gui

So far this follows "Personally, I think that we should initially focus on group 4 (new to DVC but not VS Code)," (from https://github.com/iterative/dvc.org/issues/3717#issuecomment-1181105300).

Closes #3717

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Possibly to-do here:

> 1. Pre-existing DVC users, who use VS Code
> 2. Pre-existing DVC users, who have never used VS Code
> ...
> DVC docs should be driving users in group 1 & 2 towards some kind of easy trial of the extension

_From https://github.com/iterative/dvc.org/issues/3717#issuecomment-1181105300_

**Comment by shcheklein** [comment]
@rogermparent cool reports! :) 😎 @iterative/cml can you take a look if we can simplify the action that Roger put together to use CML?

**Comment by shcheklein** [comment]
@rogermparent can we use ❌ and ✅ instead of PASS / FAIL?

File: content/docs/command-reference/plots/index.md [hunk 1]
```diff
@@ -44,13 +44,18 @@ DVC can work with two types of plots files:
 1. Data series files, which can be JSON, YAML, CSV or TSV.
 2. Image files in JPEG, GIF, or PNG format.
 
-DVC generates plots as static HTML webpages that can be open with a web browser.
-They can also be saved as SVG or PNG image files from the browser.
+DVC generates plots as static HTML webpages you can open with a web browser
+(they can be saved as SVG or PNG image files from there). You can also visualize
+DVC plots from the VS Code Extension, which includes a special [Plots Dashboard]
```
Comment (shcheklein) [inline content/docs/command-reference/plots/index.md:49]: let's put a link to the marketplace?

**Review by shcheklein — APPROVED** [review]

**Comment by shcheklein** [comment]
Are there any other places? (plots, table of experiments, even data?)

File: content/docs/start/data-management/metrics-parameters-plots.md [hunk 2]
```diff
@@ -132,7 +132,10 @@ Modifying stage 'evaluate' in 'dvc.yaml'
 
 ```
 
-Now let's view the plots:
+Now let's view the plots. You can run `dvc plots show` on you Terminal (shown
```
Comment (dberenbaum) [inline content/docs/start/data-management/metrics-parameters-plots.md:135]:
```suggestion
Now let's view the plots. You can run `dvc plots show` on your Terminal (shown
```

Minor question out of curiosity: why capitalize Terminal?

File: content/docs/start/experiments/visualization.md [hunk 3]
```diff
@@ -27,12 +27,11 @@ actual,predicted
 > We added a [loop] comparing the results to generate this file from the
 > predictions.
 
-[loop]:
-  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L123
-[confusion matrix]: https://en.wikipedia.org/wiki/Confusion_matrix
-
 Running the experiment with `dvc exp run` will produce `plots/confusion.csv`.
-Use `dvc plots show` to present it as an HTML file, and open it in the browser:
+You can run `dvc plots show` on you Terminal to generate and embed them in an
```
Comment (dberenbaum) [inline content/docs/start/experiments/visualization.md:31]:
```suggestion
You can run `dvc plots show` on your Terminal to generate and embed them in an
```

**Comment by dberenbaum** [comment]
Looks great! I'm sure we could do more, but this is a really nice start. Thanks @jorgeorpinel!

File: content/docs/start/data-management/metrics-parameters-plots.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md]: Good Q. Not sure why it was capitalized here, changed.

File: content/docs/command-reference/plots/index.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/plots/index.md:49]: Link added.

**Pull request merged**